rule_schema_v2.atd

```diff
@@ -410,6 +410,8 @@ type metavariable_cond = {
   ?type_ <json name="type">: string option;
   ?types: string list option;
 
+  ?kind: string option;
```

Comment:
please make it an enum, or at least add in comment the kind of strings that are possible here.
Also how do you translate those metavariable-name: in the new syntax? just metavariable_cond with kind?

Reply:
Also maybe add a comment before saying it's related to an experimental feature,
metavariable-name: ...

Reply:
created a separate PR for updating the comment.

> Also how do you translate those metavariable-name: in the new syntax? just metavariable_cond with kind?

Yes. like metavariable_cond with type for metavariable-type.